doubly_linked_list/doubly_linked_list.py

```diff
@@ -151,4 +151,12 @@ def delete(self, node):
       self.length -= 1
     
   def get_max(self):
```

Comment:
The get max unit tests failed because your add_to_tail function does not include the logic that converts the previous tail to the head when it was also the head initially.